packages/app/src/domain/layout/municipality-layout.tsx

```diff
@@ -29,9 +29,9 @@ export const municipalPageMetricNames = [
   'difference',
 ] as const;
 
-export type MunicipalPageMetricNames = typeof municipalPageMetricNames[number];
+export type GMPageMetricNames = typeof gmPageMetricNames[number];
```

Comment:
Please write acronyms as normal words. We do that in other places too

```suggestion
export type GmPageMetricNames = typeof gmPageMetricNames[number];
```